dea_conflux/__main__.py

```diff
@@ -415,6 +415,71 @@ def get_ids(product, expressions, verbose, s3):
     return 0
 
 
+@main.command()
+@click.argument('name')
+@click.option(
+    '--timeout', type=int,
+    help='Visibility timeout in seconds',
+    default=18 * 60)
+@click.option(
```

Comment:
If deadletter is none, we make a deadletter queue? Otherwise we use the specified one? Unclear, please clarify in the doc

Reply:
In my current plan/setup, the deadletter queue creation come from AWS console. 

How about this check:`we check the deadletter's ARN from its name. if None, throw exception and stop the application`? Then the `make queue` step in Airflow will crash and user will know we cannot create a queue and connect it to a deadletter queue.

Reply:
Hmmmm, I imagine a scenario where we run two conflux at once. Each queue should have its own deadletter, right? Oh, unless you mean make deadletter queue inside airflow with `aws` cli?

Reply:
No problem. I believe I do not realise this case. Let me clean-up the AWS/Airlfow/Conflux sections in another PR.